TreeDB/collections/vector_index_column_graph.go

```
@@ -0,0 +1,213 @@
+	cfg := catalog.meta.Options.ColumnStore
+	if cfg == nil || !cfg.Enabled || cfg.ActiveManifest == nil {
+		setColumnGraphUnavailable(&status, vectorIndexFallbackColumnGraphPhysicalMissing)
+		return nil, status, nil
+	}
+	if status.RootID == 0 {
+		setColumnGraphUnavailable(&status, vectorIndexFallbackColumnGraphManifestMissing)
+		return nil, status, nil
+	}
+	if err := validateColumnStoreCatalogRoot(snap, catalog); err != nil {
+		setColumnGraphUnavailable(&status, vectorIndexFallbackColumnGraphManifestInvalid)
+		return nil, status, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 98) but it returns nil

(nilerr)